### PR TITLE
Fix/aut 3654/import test with item class search

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Add item class destination searchable select input.

Screenshot 2024-05-22 at 10 31 17

https://github.com/oat-sa/extension-tao-test/assets/16231681/8b65dbd2-353e-45cd-ac70-2113c4d75786

### CODE REVIEW

File: views/js/controller/testImport/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+/**
+ * @author Martin Nicholson <martin@taotesting.com>
```
Comment (tikhanovichA) [inline views/js/controller/testImport/index.js:20]: Is he aware :)

File: views/js/util/provider/itemClassProvider.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+/**
+ * @author Martin Nicholson <martin@taotesting.com>
```
Comment (tikhanovichA) [inline views/js/util/provider/itemClassProvider.js:20]: Should be Bart here :)

Comment (viktar-dzmitryieu-tao) [reply views/js/util/provider/itemClassProvider.js:20]: yeah, must be Bart

Comment (bartlomiejmarszal) [reply views/js/util/provider/itemClassProvider.js:20]: I just removed.

File: views/js/util/form/itemClassSelectorForm.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+         * @param {Object} options
+         * @param {jQuery} options.$filterContainer
+         * @param {jQuery} options.$inputElement
+         * @param {taskQueueButton} options.taskButton - button which submits the form
+         * @param {Function} options.dataProvider - provider function which returns a Promise
+         * @param {String} options.inputPlaceholder
+         * @param {String} options.inputLabel
+         * @returns {filter} component which manages the form input
+         */
+        createSelectorInput({
```
Comment (Karol-Stelmaczonek) [inline views/js/util/form/itemClassSelectorForm.js:39]: Can you please reformat whole file? My eyes hurt.

File: views/js/util/provider/itemClassProvider.js (hunk 2 above)
Comment (Karol-Stelmaczonek) [reply views/js/util/provider/itemClassProvider.js:20]: Its still there ... and date is wrong :|

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/js/util/form/itemClassSelectorForm.js (hunk 3 above)
Comment (bartlomiejmarszal) [reply views/js/util/form/itemClassSelectorForm.js:39]: Better?

**Pull request merged**